CommandScreen.h

```diff
@@ -0,0 +1,24 @@
+/* Do not edit this file. It was automatically generated. */
+
+#ifndef HEADER_CommandScreen
+#define HEADER_CommandScreen
+
+#include "InfoScreen.h"
+
+typedef struct CommandScreen_ {
+   InfoScreen super;
+} CommandScreen;
+
+extern InfoScreenClass CommandScreen_class;
+
+CommandScreen* CommandScreen_new(Process* process);
+
+void CommandScreen_delete(Object* this);
```

Comment:
Do these four functions need to be global ones, can't they be static ones?

Reply:
I guess that's possible? I just followed the convention in OpenFilesScreen.h and EnvScreen.h, though.

Reply:
OK, I gave it a try, and found that I also have to:
1. remove the static functions from CommandScreen.h due to unused-function warning
2. subsequently move up the static functions in CommandScreen.c to ensure they're defined before use.

See the yet to be merged commits here: https://github.com/ryenus/htop/compare/4b02f0d...static-fn

BTW, `CommandScreen_delete` can't be static as it's called from `Action.c`. 

@cgzones How about that? Should we proceed from here?

Reply:
Sorry, I missed that `CommandScreen_delete()` is getting called from `Action.c`.
The static-fn branch looks good to me.
Please squash all the commits into a single one.

Reply:
done